CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+## v1.38
+
 Remember to bring your dependencies up to date with `pip install -r requirements.txt` when updating to this version!
```

Comment:
just a comment aside: these should probably be changed to use `./scripts/venvinstall.sh` in another PR